MAUI/ImageEditor/zooming.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Zooming in .NET MAUI ImageEditor Control | Syncfusion
+description: Learn here all about how to Zooming in Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Zooming in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
+
+The following properties are used for zooming feature of the image editor control:
+
+* EnableZooming
+* ZoomLevl
+* Maximum ZoomLevel
+
+## Enable zooming
+
+To enable or disable the zooming functionality, set the value of the [`AllowZoom`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AllowZoom) property to true or false. By default, the [`AllowZoom`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AllowZoom) value is set to `true`.
+
+{% tabs %}
+
+{% highlight XAML %}
+  <ContentPage 
+            . . .
+            <imageEditor:SfImageEditor Source="image.jpeg" AllowZoom="False"/>
+
+   </ContentPage>
+     
+{% endhighlight %}
+
+{% highlight C# %}
+   
+     SfImageEditor imageEditor = new SfImageEditor();
+     imageEditor.Source = "image.jpeg";
+     imageEditor.AllowZoom = false;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Zoom level
+
+Programmatically, you can zoom the loaded image without any interaction by setting a value to [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property. 
+
+{% tabs %}
+
+{% highlight XAML %}
+  <ContentPage 
+            . . .
+            <imageEditor:SfImageEditor Source="image.jpeg" ZoomLevel="2"/>
+
+   </ContentPage>
+     
```

Comment:
Add different way to load image source in Getting started title as discussed

Reply:
Added a different way to load image source